workspaces/tests/deploy_project.rs

```diff
@@ -1,23 +1,22 @@
 #![cfg(feature = "unstable")]
 #![recursion_limit = "256"]
 use test_log::test;
-use workspaces::prelude::*;
 
 #[test(tokio::test)]
 async fn test_dev_deploy_project() -> anyhow::Result<()> {
     let worker = workspaces::sandbox().await?;
     let wasm = workspaces::compile_project("./tests/test-contracts/status-message").await?;
     let contract = worker.dev_deploy(&wasm).await?;
 
-    let _res = contract
-        .call(&worker, "set_status")
+    contract
+        .call("set_status")
```

Comment:
Curious why this isn't required with `must_use`. Isn't the point of this PR to disallow not using the result?